aep/general/0158/aep.md.j2

```diff
@@ -167,48 +159,17 @@ now is only getting the first 50 (and does not know to advance pagination).
 Even if the API set a higher default limit, such as 100, the user's collection
 could grow, and _then_ the code would break.
 
-For this reason, it is important to always add pagination to operations
-returning collections _up front_; they are consistently important, and they can
-not be added later without causing problems for existing users.
+For this reason, it is important to always add pagination to RPCs returning
+collections _up front_; they are consistently important, and they can not be
+added later without causing problems for existing users.
 
 **Warning:** This also entails that, in addition to presenting the pagination
 fields, they **must** be _actually implemented_ with a non-infinite default
 value. Implementing an in-memory version (which might fetch everything then
 paginate) is reasonable for initially-small collections.
 
-## Implementation
+[list]: ./list
 
-Page tokens **should** be versioned independently of the public API, so that
```

Comment:
Why remove this section?  Do you think it's not useful guidance?

Reply:
It looks like this section was removed (or maybe never present?) in the Google AIP.

Reply:
Discussed live: we'll drop this section but file a ticket to keep track of potentially adding it back.